modules/dpu-enable-dpu-hw-offloading.adoc

```diff
@@ -0,0 +1,330 @@
++
+<4> Optional: The priority is an integer value between 0 and 99. A smaller value receives higher priority. For example, a priority of 10 is a higher priority than 99. The default value is 99.
++
+<5> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For a Mellanox NIC, the number of VFs cannot be larger than 128.
++
+<6> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
++
+<7> The vendor hexadecimal code of the SR-IOV network device. Vendor id `15b3` is for Mellanox devices.
++
+<8> The device hexadecimal code of the SR-IOV network device. For example, `a2d6` is the device ID for a Bluefield-2 DPU device.
++
+<9> An array of one or more physical function (PF) names for the device.
++
+<10> An array of one or more PCI bus addresses for the PF of the device. Provide the address in the following format: `0000:02:00.1`.
++
+.. Create the SriovNetworkNodePolicy object:
++
+[source,terminal]
+----
+$ oc create -f SriovNetworkNodePolicy.yaml
+----
+
+. Create a Cluster Network Operator (CNO) ConfigMap in the tenant cluster setting the mode to `dpu-host`. Save the YAML in the `sriovdpuconfigmap.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: dpu-mode-config
+    namespace: openshift-network-operator
+data:
+    mode: "dpu-host"
+immutable: true
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f sriovdpuconfigmap.yaml.yaml
+----
+
+. Set the environment variable `OVNKUBE_NODE_MGMT_PORT_NETDEV` for each DPU host. Save the YAML in the `setenvovnkube.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-overrides
+  namespace: openshift-ovn-kubernetes
+data:
+  x86-worker-node0: |
+    OVNKUBE_NODE_MGMT_PORT_NETDEV=ens1f0v0
+----
+
+.. Run the following command :
++
+[source,terminal]
+----
+$ oc create -f setenvovnkube.yaml
+----
+
+. Label the DPUs in the tenant cluster. Run the following command :
++
+[source,terminal]
+----
+$ oc label node <NODENAME> network.operator.openshift.io/dpu-host=
+----
+
+== Final infrastructure cluster configuration
+
+Run the following steps against the infrastructure cluster.
+
+. Add the `kubeconfig` of the tenant cluster as a secret.
++
+[source,terminal]
+----
+$ oc create secret generic tenant-cluster-1-kubeconf --from-file=config=/root/kubeconfig.tenant
+----
+
+. Add the per node configuration override for the `ovnkube-node`.
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: env-overrides
+  namespace: default
+data:
+  worker-bf: |
+    TENANT_K8S_NODE=x86-worker-node0
+    DPU_IP=192.168.111.29
+    MGMT_IFNAME=pf0vf0
+----
+
+. Update the `OvnkubeConfig` CR by adding the `kubeConfigFile` field. Wait until the ovnkube pods are created.
```

Comment:
In the DPU network operator, there're some status conditions that can be used to check the operator's working status. https://github.com/openshift/dpu-network-operator/pull/8